suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -302,6 +302,8 @@ tests:
 
   - test:
       desc: Perform flatten operations while changing the image feature
+      config:
+        timeout_value: 120 # rbd_op_thread_timeout
```

Comment:
Instead of taking the key here as timeout_value and again writing another key rbd_op_thread_timeout in the code, you can directly take the key to be **rbd_op_thread_timeout** itself here right and parse both key and value and use it in code?

Reply:
Done.